### PR TITLE
Feature/56 local storage helper and interceptors

### PR DESCRIPTION
**Pull request by tanmaythole**
Create a local storage helper and interceptors in axios

**Summary**
This PR adds the utils file for localStorage and its methods. And adds the interceptors in the axis instance file to generate a new access token using the refresh token.

**Ticket Link**
https://trello.com/c/xfznWjbN/56-local-storage-helper-and-interceptors

### CODE REVIEW

File: src/axios.js [hunk 1]
```diff
@@ -1,22 +1,94 @@
...
+            error.response.statusText === "UNAUTHORIZED"
+        ) {
+            const updatedToken = await refreshExpiredToken();
+            originalRequest.headers["Authorization"] = `Bearer ${updatedToken}`;
+            return axiosInstance(originalRequest);
+        }
+    }
+);
+
+const refreshExpiredTokenClosure = () => {
```
Comment (ChinmaySonawane) [inline src/axios.js:68]: can we add comments here? For a better understanding of the code.

File: src/utils/storage.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+export const setLocalStorage = (key, value) => {
```
Comment (ChinmaySonawane) [inline src/utils/storage.js]: Name of file should be localstorage.js

File: src/axios.js (hunk 1 above)
Comment (tanmaythole) [reply src/axios.js:68]: Yeah sure, I will do that

File: src/utils/storage.js (hunk 2 above)
Comment (tanmaythole) [reply src/utils/storage.js]: ok

File: src/utils/storage.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    try {
+        localStorage.setItem(key, value);
```
Comment (ChinmaySonawane) [inline src/utils/storage.js]: are we just setting primary data types here?

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**